adafruit_wiznet5k/adafruit_wiznet5k_dhcp.py

```diff
@@ -361,9 +365,9 @@ def request_dhcp_lease(
                     print("* DHCP: Parsing OFFER")
                 msg_type, xid = self.parse_dhcp_response(self._response_timeout)
                 if msg_type == DHCP_OFFER:
-                    # use the _transaction_id the offer returned,
-                    # rather than the current one
-                    self._transaction_id = self._transaction_id.from_bytes(xid, "l")
+                    # # use the _transaction_id the offer returned,
+                    # # rather than the current one
+                    # self._transaction_id = self._transaction_id.from_bytes(xid, "l")
```

Comment:
why did you comment this out?

Reply:
During DHCP, I saw that transaction ID changed. I think, no reason to change transaction id during single DHCP operation.

Reply:
Let's remove this